software/speller/data_collection_platform/dcp/mp/shared.py

```diff
@@ -5,5 +5,14 @@
 # creating a manager that will allow processes to be communicated to from the flask app
 # the manager will control when processes can pull from the bci device
 manager = Manager()
-# {process_id: {character, phase, frequency, q, config_id, collecting}}
+# {process_id: {
+#   character : str, 
+#   phase: float, 
+#   frequency: float, 
+#   q: Queue, 
+#   config_id: int,
+#   bci_config: str,
+# , state: str (Could be either start, ready, collect, stop)
```

Comment:
what is the purpose of `start` here? I understand that this is the default value, however should we set a different word for the default value? because `start` is not used in `stream_bci()`.

Reply:
we can set a different value if you want, feel free to propose something else if you think start is not clear